app/helpers/__init__.py

```diff
@@ -0,0 +1,24 @@
+import os
+import logging
+import logging.config
+
+import yaml
+
+logger = logging.getLogger(__name__)
+
+
+def get_logging_cfg():
+    logger.error(os.getcwd())
```

Comment:
Remove this logger.error here or change it to a proper debug message

Reply:
done